modules/user_accounts/php/edit_user.class.inc

```diff
@@ -1218,6 +1224,71 @@ class Edit_User extends \NDB_Form
         return $permission;
     }
 
+    /**
+     * This function returns a boolean representing whether or not
+     * to display examiner fields to a user that is editing an account.
+     * Examiner fields SHOULD be displayed in the following cases:
+     *      (1) Editor has 'examiner_multisite' permission
+     *          (which allows to add / certify examiners across all sites).
+     *      (2) Editor has 'examiner_view' permission
+     *          (which allows them to add / certify examiner at their own site)
+     *              AND  either:
+     *                  i) The user is from same site as editor,
```

Comment:
please add explicitly that:

The editor does not need to have ALL the sites of the editee (generally thats how it works in other places of loris) because the editor will not see on the front end any sites that he/she does not belong to